package.json

```diff
@@ -135,6 +135,8 @@
     "majo": "^0.6.2",
     "mockdate": "^2.0.1",
     "moment-timezone": "^0.5.5",
+    "node-glob": "^1.2.0",
```

Comment:
we're using `require('glob)` is it the same one

Reply:
should be "glob": "^7.1.2": https://github.com/isaacs/node-glob/blob/master/package.json#L3